src/main/java/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategyTransformer.kt

```diff
@@ -953,7 +953,9 @@ internal class ManagedStrategyTransformer(
      * when a continuation is cancelled by lincheck
      */
     private inner class CancellabilitySupportMethodTransformer(methodName: String, mv: GeneratorAdapter) : ManagedStrategyMethodVisitor(methodName, mv) {
-        private val isCancel = className == "kotlinx/coroutines/CancellableContinuationImpl" && methodName == "cancel"
+        private val isCancel =
+            className == "kotlinx/coroutines/CancellableContinuationImpl" && methodName == "cancel" ||
+            className == "kotlinx/coroutines/DispatchedTask" && methodName == "cancelCompletedResult"
```

Comment:
That still is not what you want.
`DispatchedTask.cancelCompletedResult` is an empty method, and adding ignored section here will not have any effect.
Probably you want to check whether `DispatchedTask` is a superclass of the `className`.

Another way is to remove `CancellabilitySupportMethodTransformer` at all, but add `strategy.enterIgnoredSection` and `strategy.leaveIgnoredSection` invocation in `cancelByLincheck` method

Reply:
Guess, I should just check the same method but for `CancellableContinuationImpl`